changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** PayFast Changelog ***
 
+= 1.7.1 - 2025-mm-dd =
+* Add - Added support for multiple subscriptions purchased in one order.
```

Comment:
Changelog is updated when we put together the release PR so no need to update that here